src/index.css

```diff
@@ -0,0 +1,31 @@
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body{
+  font-family: 'Roboto Condensed', sans-serif;
+}
+
+:root{
+  --white:white;
+  --black :black ;
+  --black-light :rgb(37, 37, 37) ;
+  --grey-light:rgb(204, 195, 195) ;
+  --grey:grey ;
+  --white:white ;
+  --pink-dark:#a30b22 ;
+  --pink-primary1:#EC0023 ;
+  --pink-primary2:#DC1F3B ;
+  --pink-primary3:#f34d66 ;
+  --pink-sec1:#CC3E53 ;
+  --pink-sec2:#D55C70;
+  --pink-sec3:#DB7084;
+  --pink:#E18397;
+  --pink2:#EDADBE;
+  --pink-light1:#f5bbce;
+  --pink-light2:#F9D5E5;
```

Comment:
For consistency, either use rgb or hex code

Reply:
I have updated it,thnks for the review